NxMap.lua

```diff
@@ -6956,7 +6956,9 @@ function Nx.Map:UpdateOverlay (mapId, bright, noUnexplored)
 
 				local wx, wy = self:GetWorldPos (mapId, (oX + bX * TILE_SIZE_WIDTH) / layerInfo.layerWidth * 100, (oY + bY * TILE_SIZE_HEIGHT) / layerInfo.layerHeight * 100)
 				
-				if string.find(oName, "_") and bX == 0 and bY == 0 then
+				local foundUnderscore = string.find(oName, "_")
+				
+				if foundUnderscore == 1 and bX == 0 and bY == 0 then
```

Comment:
I had to change this logic, because `if string.find(oName, "_")` triggered on the `hub_b` zone from zereth mortis. AFAIK this should only check if the first character is `_`.